libraries/AP_Proximity/AP_Proximity_MAV.cpp

```diff
@@ -132,13 +132,31 @@ void AP_Proximity_MAV::handle_msg(const mavlink_message_t &msg)
 
             // iterate over proximity sectors
             for (uint8_t i = 0; i < PROXIMITY_NUM_SECTORS; i++) {
-                float angle_diff = fabsf(wrap_180(_sector_middle_deg[i] - mid_angle));
-                // update distance array sector with shortest distance from message
-                if ((angle_diff <= (PROXIMITY_SECTOR_WIDTH_DEG * 0.5f)) && (packet_distance_m < _distance[i])) {
-                    _distance[i] = packet_distance_m;
-                    _angle[i] = mid_angle;
-                    sector_updated[i] = true;
+                // update distance array sector with shortest distance
+                // from message
+                const float angle_diff = wrap_180(_sector_middle_deg[i] - mid_angle);
+                if (abs(angle_diff) > PROXIMITY_SECTOR_WIDTH_DEG*0.5f) {
```

Comment:
This is meant to be an "abs" instead of "fabsf"?

Reply:
Fixed.